src/Discord.Net.Core/Entities/Interactions/MessageComponents/SelectMenuDefaultValue.cs

```diff
@@ -0,0 +1,40 @@
+namespace Discord;
+
+/// <summary>
+///     Represents a default value of an auto-populated select menu.
+/// </summary>
+public struct SelectMenuDefaultValue
+{
+    public ulong Id { get; }
```

Comment:
can just be a readonly field

Reply:
The `struct` itself is `readonly` now, so no need to mark the field as one